diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -53,6 +53,11 @@ enum class Warnings(private val canBeAutoCorrected: Boolean, private val warn: S
     HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE(false, "files that contain multiple or no classes should contain description of what is inside of this file"),
     HEADER_NOT_BEFORE_PACKAGE(true, "header KDoc should be placed before package and imports"),
     COMMENTED_OUT_CODE(false, "you should not comment out code, use VCS to save it in history and delete this block"),
+    WHITESPACE_IN_COMMENT(true, "there should be a white space at the begining of the comment"),
```

Comment:
may be we can merge these inspections somehow, at least for spaces and for newlines?

Reply:
@aktsay6 yes, I think it makes sense to merge `WHITESPACE_IN_COMMENT` and `SPACE_BETWEEN_COMMENT_AND_CODE` into a single warning.

Reply:
And I think it will make sense to implement `COMMENT_NEW_LINE_ABOVE` near `BLANK_LINE_AFTER_KDOC`, maybe even merge them in a single inspection that there should be one blank line before and no blank lines after KDoc.